Sources/SwiftRuntimeHooks/include/SwiftRuntimeHooks.h

```diff
@@ -0,0 +1,11 @@
+#ifndef SwiftRuntimeHooks_hpp
```

Comment:
```suggestion
#ifndef SwiftRuntimeHooks_h
```
?

And this macro doesn't look very "collision-safe".